backstory/src/main/java/com/google/sps/servlets/PromptManager.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//s
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.sps.servlets;
+
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+public final class PromptManager {
+
```

Comment:
If you get a chance, please add some documentation about the class itself (what it represents, and how the internal rep [keywords] is used to represent it), even if it's trivial, just to follow the style guide.

Reply:
Done